proto/oracle/v1/query.proto

```diff
@@ -256,14 +256,22 @@ message QueryRequestPriceRequest {
 // QueryRequestPriceResponse is response type for the Query/RequestPrice RPC
 // method.
 message QueryRequestPriceResponse {
```

Comment:
Request must be a list of symbols?

Reply:
![image](https://user-images.githubusercontent.com/26691764/119829180-2cb9c780-bf25-11eb-8cdf-eab33efdfee6.png)

Reply:
And response must be a list of Price struct

Reply:
Fixed, but response will be an object with a field, which is a list of price results.